openpype/modules/kitsu/plugins/publish/validate_kitsu_intent.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+import gazu
+
+
+class ValidateKitsuIntent(pyblish.api.ContextPlugin):
+    """Validate Kitsu Status"""
+
+    order = pyblish.api.ValidatorOrder
+    label = "Kitsu Intent/Status"
+    # families = ["kitsu"]
+    optional = True
+
+    def process(self, context):
+
+        publish_status = context.data.get("intent", {}).get("value")
+        if not publish_status:
+            self.log.info("Status is not set.")
+
+        kitsu_status = gazu.task.get_task_status_by_short_name(publish_status)
```

Comment:
Intent is meant as "Intent of publishing" not status of task.

Reply:
I don't get your remark.

Reply:
You mean, when intent is not `None`, we must avoid updating the status in Kitsu, and when it's `None`, change the status to `Waiting for Approval`?

Reply:
In our case, this intent code seems out of topic, and status creation must be moved to `IntegrateKitsuNote`.

Reply:
I've made the relevant changes

Reply:
> In our case, this intent code seems out of topic, must be deleted, and status change must be moved to IntegrateKitsuNote.

That's correct. `Intent` is technically a unified comment thing that is requested by certain vendors and studios. For instance when artist publishes first version of a comp it might have status `for review` but `intent` might just be `tech validation`.... later on v03 would  still go to `for review` with intent `wip` , and at the end v06 might be `for review` and intent `for final`... or similar. intents are of course arbitrary in each studio, but most importantly they are static data on a version and don't change with status updates